docs/dev-standards/secure-sdlc.md

```diff
@@ -0,0 +1,162 @@
+# Secure Development Lifecycle (SDLC)
+
+## Introduction
+
+Security **MUST** be embedded throughout the software development lifecycle. Teams **MUST** take a DevSecOps approach, treating security and operations as intrinsic to the development process. This ensures that risks are identified and mitigated early, security controls are continuously validated, and secure practices are consistently applied from planning through to deployment and maintenance.
+
+## Policy Framework
+
+This standard implements the [NIST Secure Software Development Framework (SSDF)](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-218.pdf) practices within UKHSA's development processes. Teams **MUST** follow these requirements as the minimum baseline for secure development, with a risk-based approach that scales security controls appropriate to system criticality and data sensitivity.
+
+## Guidance
+
+### Process Requirements
+
+- Teams **MUST** follow the [UKHSA SDLC](../sdlc.md) and integrate security activities into each phase aligned with NIST SSDF practices.
```

Comment:
This link looks broken now this file has been moved to the top level. Some automated link checking would be beneficial!
```suggestion
- Teams **MUST** follow the [UKHSA SDLC](./sdlc.md) and integrate security activities into each phase aligned with NIST SSDF practices.
```